sql/V007__messages_create.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS messages(
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    sendtime TEXT NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    content TEXT NOT NULL,
+    user TEXT NOT NULL,
+    discriminator INTEGER NOT NULL,
+    channel INTEGER NOT NULL,
+    server TEXT NOT NULL
+);
```

Comment:
Can you add indices on `messages(user)`, `messages(channel)`, and `messages(server)` ?